app/src/validators/submission.ts

```diff
@@ -33,7 +33,7 @@ const schema = {
       contactEmail: emailJoi,
       contactPreference: Joi.string().min(0).max(255).allow(null),
       projectName: Joi.string().min(0).max(255).allow(null),
-      projectDescription: Joi.string().min(0).max(255).allow(null),
```

Comment:
`min(0)` most likely required here. Form fails to save after remove existing text and saving with a blank text area.